en/mavgen_python/README.md

```diff
@@ -117,12 +141,15 @@ Connection type | Connection string
 --- | ---
 Linux computer connected to the vehicle via USB | /dev/ttyUSB0
 Linux computer connected to the vehicle via Serial port (RaspberryPi example) | /dev/ttyAMA0 (also set baud=57600)
-SITL connected to the vehicle via UDP | 127.0.0.1:14550 or udp:localhost:14550
-SITL connected to the vehicle via TCP | tcp:127.0.0.1:5760
+SITL connected to the vehicle via UDP | 127.0.0.1:14540 or udp:localhost:14540
```

Comment:
It's not SITL itself. It's just that this port is offered for MAVROS or the SDK while 14550 is used by QGC usually.

Reply:
Thanks @julianoes .

These are pretty standard UDP ports for GCS and developer APIs (used by both ArduPilot and PX4). 

Are you saying these strings would be better written as: 
- Offboard API connecting to the vehicle via UDP
- GCS connecting to the vehicle via UDP
- Offboard API connecting to the vehicle via TCP

Reply:
It's not limited to offboard. You could write `SITL port (usually used by MAVROS or SDK)`.

Reply:
For now, by default, it's only supported/enabled in SITL, correct.

That depends on what offboard means. If you refer to as on another board (usually) then yes but if it means the offboard control mode, that's something else.